src/client/runner/automation/options/drag.js

```diff
@@ -0,0 +1,9 @@
+import MouseActionOptions from './mouse-action.js';
+
+export default class DragOptions extends MouseActionOptions {
+    constructor () {
+        super();
+
+        this.destination = null;
```

Comment:
We should describe the structure of automation options clearly. I would like to specify the `destination` option here. 
It may be a dom-element or { dragOffsetX, dragOffsetY }.So, we can create three fileds: `destinationElement`, `destinationOffsetX`, `dedstinationOffsetY`,
 or maybe `destinationElement`, `destinationOffsets: {x, y}`
or something else if you have a better idea